### PR TITLE
Update TaylorExpansionMoments_doc.i.in

### PR DESCRIPTION
**Pull request by adutfoy**
Modified the text to make it clearer without typos.

### CODE REVIEW

File: python/src/TaylorExpansionMoments_doc.i.in [hunk 1]
```diff
@@ -11,32 +11,16 @@ limitStateVariable : :class:`~openturns.RandomVector`
...
-:math:`\uY` towards the :math:`\uX` random vectors around the mean point
-:math:`\muX`. Depending on the order of the Taylor decomposition (classically
-first order or second order), one can obtain different formulas introduced
-hereafter.
-
-As :math:`\uY=h(\uX)`, the Taylor decomposition around :math:`\ux = \muX` at
-the second order yields to:
-
-.. math::
+Assuming that :math:`\uX` has its two first moments finite and that :math:`h` is sufficiently regular, the Taylor expansion method is used to approximate the mean and variance of a random variable :math:`\uY = h(\uX)` with respect to the mean and variance of the input random vector
```
Comment (jschueller) [inline python/src/TaylorExpansionMoments_doc.i.in:14]: this line is too long

**Comment by josephmure** [comment]
For some reason CircleCI did not run. This is unfortunate because we can't see the compiled doc.

File: python/src/TaylorExpansionMoments_doc.i.in [hunk 2]
```diff
@@ -49,13 +34,29 @@ where:
...
+      <\vect{\vect{\nabla}}h(\muX) , \: \uX - \muX> = \sum_{j=1}^{n_X} \left( \frac{\partial {\uy}}{\partial {x^j}}\right)_{\ux = \muX} . \left( X^j-\muX^j \right)
+
+and
 
--
   .. math::
 
       <<\vect{\vect{\vect{\nabla }}}^2 h(\muX,\: \vect{\mu}_{X}),\: \uX - \muX>,\: \uX - \muX> = \left( \Tr{(\uX^i - \muX^i)}. \left(\frac{\partial^2 y^k}{\partial x^i \partial x^k}\right)_{\ux = \muX}. (\uX^j - \muX^j) \right)_{ijk}
 
+The Taylor expansion of order 2 of the model :math:`h` is written by:
```
Comment (josephmure) [inline python/src/TaylorExpansionMoments_doc.i.in:49]:
```suggestion
The Taylor expansion of order 2 of the model :math:`h` is:
```

**Comment by jschueller** [comment]
moved to #2163

**Pull request closed without merging**